webapp-django/crashstats/api/views.py

```diff
@@ -318,6 +319,9 @@ def model_wrapper(request, model_name):
                 content_type='application/json; charset=UTF-8'
             )
 
+        # Execution was successful, inform Google Analytics
+        track_api_pageview(request)
```

Comment:
My original intent was to not trigger a tracking on failed things. But in afterthought I guess we can still use a decorator and trigger on the response if its status_code is 200.